t/config.t

```diff
@@ -157,6 +162,28 @@ subtest 'Test configuration override from file' => sub {
         ],
         'referers parsed correctly'
     );
+    my $untracked_assets_patterns = $app->config->{'assets/storage_duration'};
+
+    ok(ref $untracked_assets_patterns eq ref {}, "failed to read config for assets/storage_duration");
+    ok(scalar %$untracked_assets_patterns, "config is empty for assets/storage_duration");
+
+    return unless (ref $untracked_assets_patterns eq ref {} and scalar %$untracked_assets_patterns);
```

Comment:
What do you gain from exiting early in case of test failure? Is there no chance later tests can reveal more information?